pkg/volume/gce_pd/gce_pd.go

```diff
@@ -250,6 +253,7 @@ func (b *gcePersistentDiskMounter) SetUpAt(dir string, fsGroup *int64) error {
 	if b.readOnly {
 		options = append(options, "ro")
 	}
+	glog.Infof("attempting to mount %s", dir)
```

Comment:
We can probably drop the verbosity of the less critical messages, like this, to v 4. i.e. let's try to avoid spamming the logs in the normal case